testpackage/small_test_carrington_github_ci.sh

```diff
@@ -86,7 +89,7 @@ flags=$(  $run_command $bin  --version |grep CXXFLAGS)
 solveropts=$(echo $flags|sed 's/[-+]//g' | gawk '{for(i = 1;i<=NF;i++) { if( $i=="DDP" || $i=="DFP" || index($i,"PF")|| index($i,"DVEC") || index($i,"SEMILAG") ) printf "__%s", $(i) }}')
 revision=$( $run_command $bin --version |gawk '{if(flag==1) {print $1;flag=0}if ($3=="log") flag=1;}' )
 
-$small_run_command $bin --version > VERSION.txt 2> $GITHUB_WORKSPACE/stderr.txt
+#$small_run_command $bin --version > VERSION.txt 2> $GITHUB_WORKSPACE/stderr.txt
```

Comment:
Not needed/done elsewhere?

Reply:
Yeah not needed. Printing out version information into CI output could increase the size quite a bit, and well, it's inside CI so we should know which version is being run :)
The files in this directory will anyway be deleted on the next CI run.